pkg/utils/kubernetes/api.go

```diff
@@ -0,0 +1,24 @@
+package kubernetes
+
+
+import (
+	"context"
+	imagev1 "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// PlatformService ...
+type PlatformService interface {
```

Comment:
If we bring both the interface and the mock implementation here, I think we should also move over the [k8s implementation](https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kube_service.go) of it.